psydac/polar/c1_linops.py

```diff
@@ -167,7 +167,7 @@ def __isub__(self, m):
     #-------------------------------------
     # Other properties/methods
     #-------------------------------------
-    def tocoo( self ):
+    def tocoo( self, *, order='C' ):
```

Comment:
As I said before, I don't understand what `order` means for a COO matrix...

Reply:
fixed